chains/edgenchain/metadata.yaml

```
@@ -0,0 +1,23 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiKey: testkey
+    apiUrl: https://edgenscan.io/api
```

can omit apikey for blockscout
```suggestion
  - apiUrl: https://edgenscan.io/api
```